### PR TITLE
I15 use wps tools

### LINKED ISSUE #15: Factor out with wps-tool
We have built a utility tool for birds called [`wps-tools`](https://github.com/pacificclimate/wps-tools). Add this package to the project and factor out code where possible.

### PR DESCRIPTION
**Pull request by eyvorchuk**
This PR solves [issue 15](https://github.com/pacificclimate/sandpiper/issues/15), which is to modify the processes to use functions from `wps-tools`. In `wps_resolve_rules.py`, logging messages have been added and replaced the comments in the `_handler` function.

### CODE REVIEW

File: sandpiper/processes/wps_resolve_rules.py [hunk 1]
```diff
@@ -2,21 +2,25 @@
...
 import requests
 import os
 from decimal import Decimal
 from p2a_impacts.resolver import resolve_rules
 from p2a_impacts.utils import get_region, REGIONS
-
-
-LOGGER = logging.getLogger("PYWPS")
+from wps_tools.utils import log_handler
+from wps_tools.io import log_level
```
Comment (sum1lim) [inline sandpiper/processes/wps_resolve_rules.py:11]: :+1:

**Review by sum1lim — APPROVED** [review]
Both pytest and jupyter lab are working great. Nice work!

**Review by nikola-rados — COMMENTED** [review]
Code looks good but it seems the logging tools are stepping on one another:
```
2020-08-12 09:58:02 INFO: wps-tools: 
2020-08-12 09:58:02 INFO: Translating variables for query
2020-08-12 09:58:02 INFO: wps-tools: Translating variables for query
2020-08-12 09:58:02 INFO: Collecting models
2020-08-12 09:58:02 INFO: wps-tools: Collecting models
2020-08-12 09:58:02 INFO: Fetching data for temp_djf_iamean_s0p_hist
2020-08-12 09:58:02 INFO: wps-tools: Fetching data for temp_djf_iamean_s0p_hist
```
Are you getting the same thing as me?

**Comment by eyvorchuk** [comment]
Yeah, it's using the [global logger in wps-tools](https://github.com/pacificclimate/wps-tools/blob/ff65e491997d6749236b3669b70d0b5d18773e32/wps_tools/utils.py#L19) since I didn't use a logger as a parameter for `log_handler`, but rather just the log level. I could modify `wps-tools` to take a logger parameter if you'd like.

**Comment by nikola-rados** [comment]
This logging module continues to give us headaches. I think it would be worth a try for sure.

**Comment by nikola-rados** [comment]
It doesn't look like the solution has worked:
```
2020-08-12 12:00:59 INFO: 
2020-08-12 12:00:59 INFO: sandpiper: 
2020-08-12 12:00:59 INFO: 2/2 variables collected
2020-08-12 12:00:59 INFO: sandpiper: 2/2 variables collected
2020-08-12 12:00:59 INFO: Evaluating parse trees
2020-08-12 12:00:59 INFO: sandpiper: Evaluating parse trees
2020-08-12 12:00:59 INFO: 3/3 rules resolved
2020-08-12 12:00:59 INFO: sandpiper: 3/3 rules resolved
2020-08-12 12:00:59 INFO: Process complete
2020-08-12 12:00:59 INFO: sandpiper: Process complete
```
So these logs are coming from the `p2a-rule-engine` and being echoed by `sandpiper`. Furthermore I don't see any of the logs that should be coming from `sandpiper` itself. We may want to try suppressing the messages coming from the `rule-engine` to ensure we are actually from `wps-tools` at all.

**Review by nikola-rados — APPROVED** [review]
Nice work sorting this out! It is good to go now.

**Pull request merged**